docs/release_process.rst

```diff
@@ -26,292 +26,34 @@ It's common that changes have been made in the previous release that need to be
 
-.. code-block:: bash
+Strings from the Perseus plugin should also be revisited. They are maintained in the repo `learningequality/kolibri-exercise-perseus-plugin <https://github.com/learningequality/kolibri-exercise-perseus-plugin>`__, but the source strings are generated and uploaded through the main ``kolibri`` project. The translated files from CrowdIn such as ``kolibri_exercise_perseus_plugin.exercise_perseus_render_module-messages.json`` are downloaded through the ``kolibri-exercise-perseus-plugin`` project. For instructions, see :ref:`crowdin`.
 
-  python build_tools/i18n/nutritionfacts_i18n.py --message UpdateNotification.upgradeMessage0124
+.. warning:: Strings exist in more places than just the Kolibri repo. Ensure we have translations for the Perseus plugin as well as other Crowdin projects such as Kolibri Server, and the Windows, Mac, Android, and Gnome apps.
 
-This will output a JSON blob like:
 
-.. code-block:: text
+Continuing the release process
+------------------------------
 
-  {
-    "ar": {
-      "link_text": "تعلم المزيد وقم بتحميله هنا",
-      "msg": "هناك إصدار جديد متاح من كوليبري.",
-      "title": "التحديث للنسخة الجديدة أصبح متاحاً"
-    },
-    "bg-bg": {
-      "link_text": "Научи повече и изтегли оттук",
-      "msg": "Налична е нова версия на Колибри.",
-      "title": "Има налични подобрения"
-    },
-    "bn-bd": {
-      "link_text": "আরও জানুন এবং সেটি এখানে ডাউনলোড করুন",
-      "msg": "কলিব্রির একটি নতুন সংস্করণ পাওয়া যাচ্ছে।",
-      "title": "আপগ্রেড উপলব্ধ"
-    },
-    "en": {
-      "link_text": "Learn more and download it here",
-      "msg": "A new version of Kolibri is available.",
-      "title": "Upgrade available"
-    },
-    "es-es": {
-      "link_text": "Descubre más y descarga aquí",
-      "msg": "Una nueva versión de Kolibri está disponible.",
-      "title": "Actualización disponible"
-    },
-    //...
-  }
+While translators are working, we will also be finishing final feature development and beginning integration testing using our suite of Gherkin stories. During this period, we'll also perform a number of internal "bug bashes" and share beta releases externally with the community for help with review and testing.
 
+.. note:: The latest beta releases are automatically deployed to `kolibri-beta.learningequality.org <https://kolibri-dev.learningequality.org/>`__.
 
-You can `redirect this output to a file <https://askubuntu.com/questions/420981/how-do-i-save-terminal-output-to-a-file>`_ (Bash) or `pipe it to the clipboard <https://stackoverflow.com/questions/1753110/>`_ (Mac)
+The remainder of the release process is documented in structured workflows in the `Kolibri release process repo <https://github.com/learningequality/kolibri-release-process/>`__.
```

Comment:
since kolibri-release-process repo is a private repo, would that be an issue for github users outside LE org? 
or is this link just for reference? Thank you!

Reply:
I'm not sure. It's private partly because Notion is private, but there's no reason it couldn't be public.

What do you think?

Reply:
added a note indicating that it's private